app/jobs/mail_moderation_reminder_job.rb

```diff
@@ -10,6 +10,6 @@ def perform(stored_mail_id)
       MailModerationMailer.reminder_for_moderation_email(moderator, stored_mail).deliver_later
     end
 
-    MailModerationReminderJob.set(wait: 24.hours).perform_later(stored_mail_id)
+    MailModerationReminderJob.set(wait: 24.hours).perform_later(stored_mail_id) unless Rails.env.development?
```

Comment:
Stops spam of reminders on development environment without a scheduler such as sidekiq